### PR TITLE
split ts functions to separate directories

### PR DESCRIPTION
**Pull request by Shell32-Natsu**
https://github.com/GoogleContainerTools/kpt/issues/1262

### CODE REVIEW

**Comment by mengqiy** [comment]
I will rebase https://github.com/GoogleContainerTools/kpt-functions-catalog/pull/147 (currently only supports golang) after this PR merges.

File: functions/ts/helm-inflator/build/helm_inflator.Dockerfile [hunk 1]
```diff
@@ -19,9 +19,6 @@ WORKDIR /home/node/app
```
[the remark this answers is not in the document]
Comment (Shell32-Natsu) [reply functions/ts/helm-inflator/build/helm_inflator.Dockerfile:19]: The name of the Dockerfiles is defined in function SDK and it only accepts `*.Dockerfile` I believe.

[the remark this answers is not in the document]
Comment (mengqiy) [reply functions/ts/helm-inflator/build/helm_inflator.Dockerfile:19]: Got it. Let's keep it as is for now.

**Review by mengqiy — APPROVED** [review]
LGTM
Given that we (including Frank) have agreed with overall structure (e.g. gen/ directory), I'm not sure if Frank want to take a look.

**Review by frankfarzan — APPROVED** [review]
Awesome!

**Pull request merged**